Source/ACE/Network/Session.cs

```diff
@@ -108,6 +110,27 @@ public void Update(double lastTick)
                 logOffRequestTime = DateTime.MinValue;
                 SendFinalLogOffMessages();
             }
+            if (LastSaveTick == 0)

```

Comment:
For sake of readability, I left this very open. The timing will need to be changed to a better interval on the lastSaveTick. The is currently no local veriable for the interval, so I will need add that. Also believe that to increase performance, the CharacterOptions may need their own delay or moved into the logoff portion of Update.